loadfile/loadfile_test.go

```diff
@@ -20,13 +22,17 @@ func Test_LoadContext(t *testing.T) {
 	testdir := filepath.Join(TestDir, "load-ctx")
 	assert.NoError(t, os.MkdirAll(testdir, os.ModePerm))
 
+	// Randomize the number to prevent collisions when running the test many times with -count
```

Comment:
"Randomize the number" doesn't say much. There wasn't even a number before. Rather, the idea is to ensure that each instance of the test has a unique file so they don't conflict.

I'm not sure why you would need to suffix both the directory and the file ... one or the other should be fine, right?

Reply:
Note: you could also let the runtime do the randomization using the `*` pattern in `os.CreateTemp()`, which might be more straightforward.

Reply:
I switched to the os functions.